coding_guidelines_c++.md

```diff
@@ -93,6 +87,49 @@ OS X:
     # or
     sudo port install cppcheck
 
+Build from scratch:
+
+    git clone https://github.com/danmar/cppcheck.git
+    cd cppcheck
+    git checkout tags/1.69
+    make PREFIX=$PWD/install/ CFGDIR=$PWD/install/cfg HAVE_RULES=yes install
+
+    # In `.bashrc` at install/bin to `PATH`:
+    export PATH=$PATH:/path/to/cppcheck/install/bin
```

Comment:
I'd rather use
```
export PATH=/path/to/cppcheck/install/bin:$PATH
```
That way, the local version would even be picked up if a global version was lying around somewhere.